apps/emqx_conf/include/emqx_conf.hrl

```diff
@@ -46,6 +46,6 @@
     )
 ).
 
--define(READONLY_KEYS, [cluster, rpc, node]).
+-define(READONLY_KEYS, [rpc, node]).
```

Comment:
It seems this is only used in cli module. 
maybe move it to the cli module so the macro is not abused by other modules.

Reply:
also rename it to `READONLY_ROOT_KEYS` and add a comment to describe that cluster.* except for cluster.link are also read-only.